src/components/tag/index.tsx

```diff
@@ -32,15 +32,15 @@ const Tag = forwardRef<HTMLDivElement, TagProps>(
       size = 'md',
       type = 'info',
       variant = 'fill',
-      icon: Icon = TagIcons[type],
+      icon: Icon = TagIcons[type][bold ? 'true' : 'false'],
```

Comment:
```
      icon: Icon = TagIcons[type][bold],
```
Wanted to have this instead of the one in `index.tsx` however it seems to give me some errors. If there's any suggestions to make it work like the above code snippet do let me know 🙇🏼